kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -71,6 +71,21 @@ namespace Testing
             rMassMatrix(0,0) = 1.0;
         }
 
+        void CalculateLumpedMassVector(
+            VectorType& rLumpedMassVector,
+            const ProcessInfo& rCurrentProcessInfo) const override
+        {
+            unsigned int system_size = 1;
+            // Initialize the lumped mass vector
+            if (rLumpedMassVector.size() != system_size) {
+                rLumpedMassVector.resize(system_size, false);
+            }
+
+            // Fill the lumped mass vector
+            const double nodal_mass = 1.0;
+            std::fill(rLumpedMassVector.begin(),rLumpedMassVector.end(),nodal_mass);
```

Comment:
```suggestion
            rLumpedMassVector = VectorType(1);
            rLumpedMassVector(0) = 1.0;
```
I think it's way more easier in this way for people reading the test.

Reply:
Ok :)